workflows/subworkflows/predict_subtract_facets.cwl

```diff
@@ -0,0 +1,110 @@
+class: Workflow
+cwlVersion: v1.2
+id: facet_subtract_per_subband
+doc: Use WSClean to predict and subtract model data per subband and per facet.
```

Comment:
This workflow does more than just run WSClean. Perhaps it is clearer to give a high-level overview.